Helper/Data.php

```diff
@@ -461,4 +466,35 @@ public function getEndpointsTo($countryCode): array
 
         return $endpoints;
     }
+
+    public function cleanText($text)
+    {
+        return preg_replace('/[(),.#!\-]/', '', $text);
+    }
+
+    public function getExpirationTimeMinutes()
+    {
+        $minutes = $this->getExpirationTime();
+
+        return !is_numeric($minutes) || $minutes < self::EXPIRATION_TIME_MINUTES_MIN
+            ? self::EXPIRATION_TIME_MINUTES_DEFAULT
+            : $minutes;
+    }
+
+    /**
+     * @return InfoFactory
+     */
+    public function getInfoModel(): InfoFactory
+    {
+        return $this->infoFactory;
+    }
+
+    public function getHeaders(): array
```

Comment:
Update headers, check: https://github.com/placetopay/prestashop-placetopay/blob/master/src/Models/PlacetoPayPayment.php#L2705

X-Source-Platform -> magento

Reply:
Extra information in header must be: 

magento2-module-payments/$this->version (origin:$domain; vr:" . {{version_woocommerce}} . ')